facebook/__init__.py

```diff
@@ -235,7 +243,7 @@ def request(
             # or it does not need `access_token`.
             if post_args and "access_token" not in post_args:
                 post_args["access_token"] = self.access_token
-            elif "access_token" not in args:
+            elif args and "access_token" not in args:
```

Comment:
I take that back, that is necessary for one of my unit tests to pass. Otherwise I get:

    Traceback (most recent call last):
      File "/Users/seawolf/repos/facebook-sdk-jke/test/test_facebook.py", line 223, in test_get_user_permissions_nonexistant_user
        facebook.GraphAPI(token).get_permissions(1)
      File "facebook/__init__.py", line 108, in get_permissions
        "{0}/{1}/permissions".format(self.version, user_id)
      File "facebook/__init__.py", line 247, in request
        elif "access_token" not in args:
    TypeError: argument of type 'NoneType' is not iterable